command/harness/dlite/dlite.go

```diff
@@ -0,0 +1,123 @@
+package dlite
+
+import (
+	"context"
+	"time"
+
+	"github.com/drone-runners/drone-runner-aws/command/config"
+	"github.com/drone-runners/drone-runner-aws/command/harness"
+	"github.com/drone-runners/drone-runner-aws/internal/drivers"
+	"github.com/drone-runners/drone-runner-aws/store"
+	"github.com/drone-runners/drone-runner-aws/store/database"
+	loghistory "github.com/drone/runner-go/logger/history"
+	"github.com/drone/signal"
+	"github.com/joho/godotenv"
+	"github.com/sirupsen/logrus"
+	"github.com/wings-software/dlite/delegate"
+	"github.com/wings-software/dlite/poller"
+	"github.com/wings-software/dlite/router"
+	"gopkg.in/alecthomas/kingpin.v2"
+)
+
+var (
+	taskInterval  = 3 * time.Second
+	taskExecutors = 100
+	tokenExpiry   = 10 * time.Hour
+)
+
+type dliteCommand struct {
+	envFile         string
+	env             config.EnvConfig
+	poolFile        string
+	poolManager     *drivers.Manager
+	stageOwnerStore store.StageOwnerStore
+}
+
+func RegisterDlite(app *kingpin.Application) {
+	c := new(dliteCommand)
+
+	c.poolManager = &drivers.Manager{}
+
+	cmd := app.Command("dlite", "starts the runner with polling enabled for accepting tasks").
+		Action(c.run)
+	cmd.Flag("envfile", "load the environment variable file").
+		StringVar(&c.envFile)
+	cmd.Flag("pool", "file to seed the pool").
+		StringVar(&c.poolFile)
+}
+
+func (c *dliteCommand) startPoller(ctx context.Context, tags []string) error {
```

Comment:
We should also delete the pools in case we receive sigterm call if reuse pool is not set. Can be taken up in next PR.